Add new commands to Ews extension (#31534)
demisto/content: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md

| EWS.MailboxAuditBypassAssociation.WhenCreated | Date | The date the mailbox audit bypass association was created. | 
| EWS.MailboxAuditBypassAssociation.WhenCreatedUTC | Date | The date in UTC format of when the mailbox audit bypass association was created. | 

### ews-rules-list
***
Get a list of all mailbox rules.

#### Base Command

`ews-rules-list`
#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| mailbox | The mailbox that contains the Inbox rule. | Required | 
| limit | Maximum number of rules to get. A value of 0 means to get all rules | Optional | 

#### Context Output
| **Path** | **Type** | **Description** |
| --- | --- | --- |
| EWS.Rule.RuleIdentity | String | The rule identity. | 
| EWS.Rule.Name | String | The rule name. | 

> | EWS.Rule.Enabled | Boolean| If the rules is enabled or not. | 

Reviewer: ```suggestion
| EWS.Rule.Enabled | Boolean| Whether the rule is enabled or not. | 
```

| EWS.Rule.Priority | String | the rule priority. | 

#### Human Readable Output

>### Results of ews-rules-list
>| Enabled | Name | Priority | RuleIdentity |
>| --- | --- | --- | --- |
>| true | CheckActionRequired	 | 1 | 1268829516541722625 |
>| true | ews phishing test	| 8 | 1845290268845146113 |

### ews-get-rule
***
Get a mailbox rule.

#### Base Command

`ews-get-rule`
#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| mailbox | The mailbox that contains the Inbox rule. | Required | 

> | identity | the id of the rule. | Required |

Reviewer: ```suggestion
| identity | The ID of the rule. | Required |
```

#### Context Output
| **Path** | **Type** | **Description** |
| --- | --- | --- |
| EWS.Rule.Rule | String | The rule identity. | 
| EWS.Rule.RuleName | String | The rule name. | 

> | EWS.Rule.IsEnabled | Boolean| If the rules is enabled or not. | 

Reviewer: ```suggestion
| EWS.Rule.IsEnabled | Boolean| Whether the rule is enabled or not. | 
```

> | EWS.Rule.Priority | String | the rule priority. |

Reviewer: ```suggestion
| EWS.Rule.Priority | String | The rule priority. |
```

| EWS.Rule.Description | String | The description of the rule. | 

> | EWS.Rule.StopProcessingRules | Boolean| If to stop processing the rule or not. | 

Reviewer: ```suggestion
| EWS.Rule.StopProcessingRules | Boolean| Whether to stop processing the rule or not. | 
```

> | EWS.Rule.IsValid | Boolean | If the rule is valid or not. | 

Reviewer: ```suggestion
| EWS.Rule.IsValid | Boolean | Whether the rule is valid or not. | 
```

#### Human Readable Output

>### Results of ews-rules-list
>| Enabled | Name | Priority | RuleIdentity | Description | IsValid	| StopProcessingRules|
>| --- | --- | --- | --- | --- | --- | ---|
>| true | CheckActionRequired | 1 | 1268829516541722625 | If the message: the sender requested any action and my name is in the To box | true| false |

### ews-remove-rule
***
Remove a mailbox rule.

#### Base Command

`ews-remove-rule`
#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| mailbox | The mailbox that contains the Inbox rule. | Required | 

> | identity | the id of the rule. | Required | 

Reviewer: ```suggestion
| identity | The ID of the rule. | Required | 
```
